### PR TITLE
Implement pazuz ticker

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Create the pazuz ticker plugin](https://trello.com/c/6jGKAvp6/671-create-the-pazuz-ticker-plugin)

File: tools/set-env-var.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const fs = require('fs')
+const os = require('os')
+const path = require('path')
+
+const setEnvVariable = (key, value) => {
+  const ENV_VARIABLES = fs.readFileSync(path.resolve(__dirname, '../.env'), 'utf-8').split(os.EOL)
+  const target = ENV_VARIABLES.indexOf(ENV_VARIABLES.find(line => line.match(new RegExp(`^${key}`))))
```
Comment (siiky) [inline tools/set-env-var.js]: Using the regex `^${key}=` would better future proof this code. With the current regex, `foo` would also match `foobar`, which is wrong, and leads to the wrong result depending on the order of the keys in the file.

File: dev/notify.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ require('es6-promise').polyfill()
 
 var notifier = require('../lib/notifier')
 var db = require('../lib/postgresql_interface')
-var psqlUrl = require('../lib/options').postgres
+const { PSQL_URL } = require('./constants')
```
Comment (siiky) [inline dev/notify.js]: Did you mean `'../lib/constants'`?

File: lib/plugins/wallet/lnd/lnd.js [hunk 3]
```diff
@@ -20,7 +21,7 @@ module.exports = {
 }
 
 function connect () {
-  return lnd.connect(options.lnd || {})
+  return lnd.connect(LND || {})
```
Comment (siiky) [inline lib/plugins/wallet/lnd/lnd.js]: Is this right? `lnd.connect()` seems to expect an object, but values of environment variables are strings.

File: lib/plugins/wallet/monerod/monerod.js [hunk 4]
```diff
@@ -5,14 +5,16 @@ const jsonRpc = require('../../common/json-rpc')
 
 const { COINS, utils } = require('lamassu-coins')
 
-const blockchainUtils = require('../../../coin-utils')
 const BN = require('../../../bn')
 const E = require('../../../error')
 const { logger } = require('../../../blockchain/common')
+const options = require('../../../options')
+
+const blockchainDir = options.blockchainDir
```
Comment (siiky) [inline lib/plugins/wallet/monerod/monerod.js]: In the file `lib/blockchain/do-volume.js` you read the blockchain dir from the environment. Is that a different variable, or should this be read from the environment as well?

File: tools/build-prod-env.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+const requiredParams = ['db-password', 'hostname']
+
+if (!_.isEqual(_.intersection(_.keys(argv), requiredParams), requiredParams)) {
+  console.error('Usage: node tools/build-prod-env.js --db-password <DB_PASSWORD> --hostname <IP>')
+  process.exit(2)
+}
+
+const ENV_SAMPLE = fs.readFileSync(path.resolve(__dirname, '../.sample.env'), 'utf-8').split(os.EOL)
+fs.writeFileSync(path.resolve(__dirname, '../.env'), ENV_SAMPLE.join(os.EOL))
```
Comment (siiky) [inline tools/build-prod-env.js]: Any special reason to read&write instead of using `fs.copyFile()` or `fs.copyFileSync()`?

**Review by siiky — CHANGES_REQUESTED** [review]

**Review by siiky — APPROVED** [review]
LGTM

**Pull request merged**